.gitignore

```diff
@@ -64,3 +64,5 @@ go.work
 
 ### Visual Studio Code IDEA ###
 .vscode
+.terraform.lock.hcl
```

Comment:
Hmmmm - per https://www.terraform.io/language/files/dependency-lock#lock-file-location, we should actually be checking this file into source control.